benchmarks/mongo-es-speed/index.js

```diff
@@ -0,0 +1,152 @@
+import { spawn } from 'child_process';
+import rawFs from 'fs';
+import { MongoClient } from 'mongodb';
+import nyanProgress from 'nyan-progress';
+import path from 'path';
+import pify from 'pify';
+
+import prepareEvents from './prepareEvents';
+import config from './config';
+
+const POINTS = config.BENCHMARK_SERIES;
+const fs = pify(rawFs);
+
+const totalMaxValue = POINTS.reduce((acc, val) => (acc + val), 0) * 2;
+const progress = nyanProgress();
+const totalEventProcessed = { value: 0 };
+const totalEventReported = { value: 0 };
+
+const progressDonePromise = progress.start({
+    total: totalMaxValue,
+    message: {
+        downloading: ['Load testing progress'],
+        finished: 'Load testing done',
+        error: 'Load testing error'
+    }
+});
+
+const INFO_TOKEN = '-------INFO-------';
```

Comment:
To avoid copy paste move `INFO_TOKEN`, `DONE_TOKEN `, `ERR_TOKEN ` to `constants` file.